### PR TITLE
Vdo.ai Bid Adapter : update to prebid version 9

### PR DESCRIPTION
**Pull request by vdo-ai-tech**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter  
- [x] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org? 
- [ ] Other

## Description of change
Bid request parameters updated and logical implementations.
Document updated: https://github.com/prebid/prebid.github.io/pull/5690

### CODE REVIEW

**Comment by patmmccann** [comment]
it appears you have a lot of duplication with the onetag adapter. Please move the visibility code in both adapters into a common utility library, probably the precentInView one

**Comment by vdo-ai-tech** [comment]
hi @patmmccann We have removed the visibility and page timing functions since they are not required anymore on our end. LMK if anything else needs to be done.

**Comment by vdo-ai-tech** [comment]
Hi @patmmccann I’d like to follow up on the status of the merge request. Is there anything I can assist with to help expedite the process?

File: modules/vdoaiBidAdapter.js [hunk 1]
```diff
@@ -31,23 +111,80 @@ export const spec = {
...
+      if (bidderRequest && bidderRequest.uspConsent) {
+        payload.usPrivacy = bidderRequest.uspConsent;
+      }
+      if (validBidRequests && validBidRequests.length !== 0 && validBidRequests[0].schain && isSchainValid(validBidRequests[0].schain)) {
+        payload.schain = validBidRequests[0].schain;
+      }
+      if (validBidRequests && validBidRequests.length !== 0 && validBidRequests[0].userIdAsEids) {
+        payload.userId = validBidRequests[0].userIdAsEids;
+      }
+      if (config.getConfig('coppa') === true) {
```
Comment (ChrisHuie) [inline modules/vdoaiBidAdapter.js:185]: Can you please access coppa from `bidderRequest.ortb2.regs.coppa` instead of config

Comment (vdo-ai-tech) [reply modules/vdoaiBidAdapter.js]: Hey @ChrisHuie , thanks for the review. we have made the necessary changes. Is there anything else I can assist with?

**Comment by vdo-ai-tech** [comment]
Hi @patmmccann, I wanted to follow up on the merge request. If there's anything I can do to help speed up the process, please let me know. Thanks!

**Comment by vdo-ai-tech** [comment]
Hi @patmmccann, I wanted to follow up on the merge request. If there's anything I can do to help speed up the process, please let me know. Thanks!

**Review by ncolletti — CHANGES_REQUESTED** [review]
please open a docs PR to update your bidder info to support gdpr, coppa, schain, userids, and floors

**Comment by vdo-ai-tech** [comment]
> please open a docs PR to update your bidder info to support gdpr, coppa, schain, userids, and floors
Bidder info updated. Please refer: https://github.com/prebid/prebid.github.io/pull/5690

**Review by ncolletti — APPROVED** [review]
thanks for the docs update, these changes look good to me!

**Pull request merged**